app/controllers/students_controller.rb

```diff
@@ -37,6 +37,9 @@ def show
       educators_index: Educator.to_index,
       access: student.latest_access_results,
       iep_documents: student.iep_documents,
+      sections: student.sections.select('sections.*, student_section_assignments.grade_numeric')
```

Comment:
This is a lot of complexity inline with serialization code, factor out into a method?

FWIW `educators_index` is sent to the browser so you can serialize just ids and join client side if that were simpler, no need to change here just FYI.  And TIL that `#select` doesn't just work as "filter a list" but also in the SQL sense to pluck fields. :)